clang/test/3C/example.c

```diff
@@ -0,0 +1,28 @@
+struct {int *a; int *ab;} *x[4], n;
+struct foo {int *a; int *b;} *y, d; 
+struct bar {int *a; int *b;}; 
+struct bar *z; 
+
+void foo() {
+    //struct {int *d;} *f;
+} 
+
+/*converts to:
+
+
+_Ptr<struct {_Ptr<int> a; _Ptr<int> ab;}> x _Checked[4] = {((void *)0)};
+struct (anonymous struct at /Users/shilpa-roy/checkedc/checkedc-clang/shilpa_issues_examples/example.c:1:1) n;
```

Comment:
Looks like this line is not what you wanted; why did it work for `x` but not `n` ?

Reply:
What happens if you have a nested `struct` ? E.g.,
```
struct { 
  struct { int *x; int *y; } pair;
  char *s;
} *p;
```